src/main/java/frc/robot/subsystems/shooter/SingleCanSparkMaxSubsystem.java

```diff
@@ -52,18 +53,23 @@ protected SingleCanSparkMaxSubsystem(String name, int motorId, double voltage, b
      */
     protected SingleCanSparkMaxSubsystem(String name, int motorId, double fowardsVoltage, double reverseVoltage,
             boolean isInverted) {
+
         super(name);
+
+        FORWARDS_VOLTAGE = fowardsVoltage;
+        REVERSE_VOLTAGE = reverseVoltage;
```

Comment:
I would prefer to merge https://github.com/Team1559/2024-Crescendo/pull/21 instead of diverging further.